scripts/fill-messages-to-influx.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+var mongoose = require('mongoose');
+var co = require('co');
+var path = require('path');
+var influxService = require(path.resolve('./modules/core/server/services/influx.server.service'));
+let fluxIn = require(path.resolve('./modules/messages/server/services/message-to-influx.server.service'));
+
+mongoose.Promise = Promise;
```

Comment:
It looks like @houndci-bot isn't respecting `env: { node: true }` from eslint. Just ignore.